test/test_marker.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+
+def sum(a, b):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'sum'](https://app.codacy.com/manual/mathieugouin/tradesim/pullRequest?prid=5529097)